src/lib/sidenav/sidenav.md

```diff
@@ -48,37 +137,42 @@ mat-sidenav {
 
 Try to avoid percent based width as `resize` events are not (yet) supported.
 
-For a fullscreen sidenav, the recommended approach is set up the DOM such that the
-`<mat-sidenav-container>` can naturally take up the full space:
+### Fixed position sidenavs
 
-```html
-<app>
-  <mat-sidenav-container>
-    <mat-sidenav mode="side" opened="true">Drawer content</mat-sidenav>
-    <div class="my-content">Main content</div>
-  </mat-sidenav-container>
-</app>
-```
-```css
-html, body, material-app, mat-sidenav-container, .my-content {
-  margin: 0;
-  width: 100%;
-  height: 100%;
-}
-```
+For `<mat-sidenav>` only (not `<mat-drawer>`) fixed positioning is supported. It can be enabled by
+setting the `fixedInViewport` property. Additionally, top and bottom space can be set via the
+`fixedTopGap` and `fixedBottomGap`. These properties accept a pixel value amount of space to add at
+the top or bottom.
 
-### FABs inside sidenav
-For a sidenav with a FAB (Floating Action Button) or other floating element, the recommended approach is to place the FAB
-outside of the scrollable region and absolutely position it.
+<!-- example(sidenav-fixed) -->
 
+### Creating a responsive layout for mobile & desktop
 
-### Disabling closing of sidenav
-By default, sidenav can be closed either by clicking on the backdrop or by pressing <kbd>ESCAPE</kbd>.
-The `disableClose` attribute can be set on `mat-sidenav` to disable automatic closing when the backdrop
-is clicked or <kbd>ESCAPE</kbd> is pressed. To add custom logic on backdrop click, add a `(backdropClick)` listener to
-`mat-sidenav-container`. For custom <kbd>ESCAPE</kbd> logic, a standard `(keydown)` listener will suffice.
-```html
-<mat-sidenav-container (backdropClick)="customBackdropClickHandler()">
-  <mat-sidenav disableClose (keydown)="customKeydownHandler($event)"></mat-sidenav>
-</mat-sidenav-container>
-```
\ No newline at end of file
+A sidenav often needs to behave differently on a mobile vs a desktop display. On a desktop, it may
```

Comment:
Might be useful to show how to make this happen.

Reply:
That's what the example linked at this point does